### ISSUE TITLE
DIB-R train.py losses

### ISSUE THREAD

**Issue by srs7** [NVIDIAGameWorks/kaolin#310]
I think there is a mismatch between the loss in the training example for 3D reconstruction from a single image using DIB-R and in the original paper implementation. In particular, silhouette (alpha_loss) and color (img_loss). 

In [DIB-R original implementation](https://github.com/nv-tlabs/DIB-R/blob/master/dib-render/test-all.py)

`            colloss = 3 * torch.mean(torch.abs(impre_vvbx3xhxw - imgt_vvbx3xhxw))
`

in [Kaolin Image Recon](https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/ImageRecon/Dib/train.py) 

`img_loss = ((image_pred - image_gt)**2).mean()
`
According to the paper is an L1 norm. 

Something similar happens with alpha_loss.
 
Is there a reason for that? Am I missing something?

**Comment by shumash** [comment]
Hi, thank you for using Kaolin! The library went through a major rewrite with the version [0.9.0](https://github.com/NVIDIAGameWorks/kaolin/releases/tag/v0.9.0), and this issue is no longer current.

Please refer to our latest documentation:
[https://kaolin.readthedocs.io/en/latest/](https://kaolin.readthedocs.io/en/latest/)

And to our DIB-R tutorial:
[examples/tutorial/dibr_tutorial.ipynb](https://github.com/NVIDIAGameWorks/kaolin/blob/master/examples/tutorial/dibr_tutorial.ipynb)
